events/datadog.go

```diff
@@ -54,7 +54,8 @@ func (e *Datadog) getGodspeed() {
 	if e.Config.Host == "" || e.Config.Port == "" {
 		gs, err = godspeed.NewDefault()
 	} else {
-		port, err := strconv.Atoi(e.Config.Port)
+		var port int
+		port, err = strconv.Atoi(e.Config.Port)
```

Comment:
I looks like this error will be logged twice. This function should probably just be refactored to handle the errors when they happen.